discovery/spec.md

```diff
@@ -830,15 +830,14 @@ Service deletions MUST be processed in the order in which they are given. Thus,
 if a Service is specified multiple times the entire request MUST fail.
 
 The follow responses are defined by this specification:
-- `200 OK` if all the specified Services were deleted successfully.
+- `200 OK` if all the specified Services were deleted successfully or are not
```

Comment:
perhaps "`200 OK` if none of the specified Services remain in the Discovery Endpoint."

Reply:
done